src/config.ts

```diff
@@ -13,6 +13,10 @@ export class Config {
     type: 'tcp' | 'udp4' | 'udp6'
     /** The path on disk where to write logs to */
     logPath?: string
+    /** the TCP Keep-Alive Interval (milliseconds) for the 'client' Connection, defaults to none */
+    clientKeepAliveInterval: number
+    /** the TCP Keep-Alive Interval (milliseconds) for the 'server' Connection, defaults to none */
+    serverKeepAliveInterval: number
```

Comment:
There need to move please, see previous comment

Reply:
ok cool updated